manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Elgato.CameraHub
+PackageVersion: 1.3.0.596
+PackageLocale: en-US
+Publisher: Elgato Systems

```

```suggestion
Publisher: Elgato
```
This must match Control Panel.